src/app/projects/states/dashboard/directives/student-task-list/student-task-list 2.coffee

```diff
@@ -0,0 +1,62 @@
+angular.module('doubtfire.projects.states.dashboard.directives.student-task-list', [])
+#
+# View a list of tasks
+#
+.directive('studentTaskList', ->
```

Comment:
Shouldn't CoffeeScript code be removed? whats happening here?

Reply:
Not part of my task, I dont why thats there. I will delete the file